fedimint-bitcoind/src/lib.rs

```diff
@@ -198,6 +198,7 @@ impl DynBitcoindRpc {
     pub fn spawn_block_count_update_task(
         self,
         task_group: &TaskGroup,
+        propose_citem_tx: watch::Sender<()>,
```

Comment:
This leaks though layers of abstractions a bit here. Could be replaced with passing a custom lambda that gets called with new count_update, but whatevs for now.

Reply:
Done.